job-dsls/src/main/resources/job-scripts/prod_rhba_stage_cpaas_artifacts.jenkinsfile

```diff
@@ -0,0 +1,100 @@
+def artifacts = [
+    [originClassifier: "maven-repository.zip", finalClassifier: "maven-repository.zip", stageCommand: "cp", info: "Copying maven repository zip"],
+    [originClassifier: "license.zip", finalClassifier: "licenses.zip", stageCommand: "cp", info: "Copying licenses zip"],
+    [originClassifier: "src.zip", finalClassifier: "sources.zip", stageCommand: "cp", info: "Copying sources zip"],
+    [originClassifier: "offliner.zip", finalClassifier: "offliner.zip", stageCommand: "mv", info: "Moving offliner zip"]
+]
+
+pipeline {
+    agent {
+        label 'kie-rhel7 && !master'
+    }
+
+    environment {
+        PRODUCT_NAME = PRODUCT_NAME.trim()
+        VERSION = VERSION.trim()
+        RCM_HOST = RCM_HOST.trim()
+        STAGING_BASE_PATH = STAGING_BASE_PATH.trim()
+        PRODUCT_NAME_LOWER_CASE = PRODUCT_NAME.toLowerCase()
+        RHBA_FOLDER_PATH = "${STAGING_BASE_PATH}/rhba/rhba-${VERSION}"
+        TARGET_FOLDER_PATH = "${STAGING_BASE_PATH}/${PRODUCT_NAME_LOWER_CASE}/${PRODUCT_NAME}-${VERSION}"
+    }
+
+    stages {
+        stage('Clean workspace') {
+            steps {
+                cleanWs()
+            }
+        }
+
+        stage('Print variables') {
+            steps {
+                println "[INFO] PRODUCT_NAME: ${PRODUCT_NAME}"
+                println "[INFO] VERSION: ${VERSION}"
+                println "[INFO] RCM_HOST: ${RCM_HOST}"
+                println "[INFO] STAGING_BASE_PATH: ${STAGING_BASE_PATH}"
+            }
+        }
+
+        stage('Clone integration-platform-tooling repository') {
+            steps {
+                sh "git clone ${INTEGRATION_PLATFORM_TOOLING_REPOSITORY} integration-platform-tooling"
+            }
+        }
+
+        stage("Create offliner zip") {
+            steps {
+                script {
+                    createOfflinerZip(RHBA_FOLDER_PATH)
+                }
+            }
+        }
+
+        stage('Stage artifacts produced by CPaaS') {
+            steps {
+                script {
+                    for (artifact in artifacts) {
+                        stageArtifact(artifact, RHBA_FOLDER_PATH, TARGET_FOLDER_PATH)
+                    }
+                }
+            }
+        }
+    }
+
+    post {
+        always {
+            cleanWs()
+        }
+    }
+}
+
+def createOfflinerZip(rhbaFolderPath) {
```

Comment:
either we move all the required global variables required by `createOfflinerZip` method like `RHBA_FOLDER_PATH`, `PRODUCT_NAME_LOWER_CASE`, `VERSION` etc... or we remove `rhbaFolderPath` argument and `RHBA_FOLDER_PATH` is directly used.